app/views/messages/_list.html.slim

```diff
@@ -14,7 +13,7 @@
             => link_to 'Edit', edit_message_path(message)
             = link_to 'Delete', message_path(message), method: :delete, data: {confirm: 'Are you sure?'}
 
-      = message.text
+      = sanitize message.text, tags: %w(p strong em u table thead tbody th tr td h4 h5 h6 pre br hr ul ol li blockquote), attributes: %w(id class style)
```

Comment:
could be moved into helper method, because we already use the same thing on different page